ndb/src/google/cloud/ndb/client.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A client for NDB which manages credentials, project, namespace."""
+
+import os
+
+from google.cloud import environment_vars
+from google.cloud.client import ClientWithProject
+from google.cloud import _helpers
+
+_DATASTORE_HOST = "datastore.googleapis.com"
```

Comment:
Why is this needed? Isn't there a constant in `google-cloud-datastore` we can use?

Reply:
There is `google.cloud.datastore.client._DATASTORE_BASE_URL`, which could be used to derive the host. It's private, and you had mentioned copying private members from Datastore rather than depending on them. We don't use a URL, since we don't support the HTTP transport, but we could derive the host from the URL and may implement that HTTP transport in a future iteration. It's not clear to me there's a single right answer here.

Reply:
Isn't it defined anywhere in the `gapic` module(s)?

Reply:
Found it in _http. Which is technically still private, but I guess I'll use that.